internal-services/catalog/file-updates-process-file-updates-task.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: task-process-file-updates
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Update files in a Git repository
+  params:
+      - name: fileUpdates
+        type: string
+        description: fileUpdates json string
+  results:
+      - name: fileUpdatesInfo
+        description: fileUpdates detailed information
+      - name: fileUpdatesState
+        description: fileUpdates state
+  workspaces:
+      - name: temp
```

Comment:
What's the workspace for? Usually it's used when you need to share data between steps or even tasks. But it doesn't seem to be happening here - you're just cloning the repos there, but they could be cloned anyway. Or is it in preparation for a new step that will create the MRs? In that case it would be ok, but I would propose something less generic for the name, e.g. `repos`.

Reply:
Also, not sure if it ok to rely in the space available in the container to fetch the repo. It might be huge depending on the case.

Reply:
> Also, not sure if it ok to rely in the space available in the container to fetch the repo. It might be huge depending on the case.

How big is the workspace now? Do you know? The PipelineRun is created by an operator and the operator provides a PV, right?

I was trying to find out what is the default size of the ephemeral storage provided for the Pods in Tekton, but couldn't find it here: https://tekton.dev/docs/pipelines/compute-resources/ It would be nice if we could compare those.

But this brings another point - I think we should definitely use shallow clones here: https://github.com/redhat-appstudio/release-service-utils/blob/main/utils/git-functions#L71 That will save a lot of space. Can you create a PR for that?

Reply:
For now I changed to /tmp. I agree with your point on the shallow clone, but If you don't mind, I prefer to open a ticket and make the changes when I am back in two weeks, so I can at least close this ticket which is there for about a month.

Reply:
I might do it still today, but just in case we take to long to close this.

Reply:
https://github.com/redhat-appstudio/release-service-utils/pull/51